functions/src/Controller/User.ts

```diff
@@ -35,4 +35,36 @@ export const editUserController = functions.https.onRequest(async (request, resp
 			response.status(404).send('Error 404: User document not found');
 		}
 	});
+});
+
+export const openApplicationsController = functions.https.onRequest(async (request, response)=> {
+	const userID: string = request.body.id;
+	const userCollection = getUserCollection();
+	const userDoc = userCollection.doc(userID);
+
+	await userDoc.get().then((docSnapshot) => {
+		if (docSnapshot.exists) {
+			void openApplications(userID);
+			response.status(200).send('Good Job');
+		}
+		else {
+			response.status(404).send('Error 404: User document not found');
+		}
+	});
+});
+
+export const closeApplicationsController = functions.https.onRequest(async (request, response)=> {
+	const userID: string = request.body.id;
+	const userCollection = getUserCollection();
+	const userDoc = userCollection.doc(userID);
+
+	await userDoc.get().then((docSnapshot) => {
+		if (docSnapshot.exists) {
+			void closeApplications(userID);
```

Comment:
I think the convention has been to remove "void" and instead handle promises like we've been doing in the services files.